dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -176,25 +176,27 @@ await foreach (Completions completions in response)
         };
     }
 
-    private static Dictionary<string, object?> GetResponseMetadata(ChatCompletions completions)
+    private static Dictionary<string, object?> GetResponseMetadata(ChatCompletions completions, int iterations)
     {
-        return new Dictionary<string, object?>(5)
+        return new Dictionary<string, object?>(6)
         {
             { nameof(completions.Id), completions.Id },
             { nameof(completions.Created), completions.Created },
             { nameof(completions.PromptFilterResults), completions.PromptFilterResults },
             { nameof(completions.SystemFingerprint), completions.SystemFingerprint },
             { nameof(completions.Usage), completions.Usage },
+            { "Iterations", iterations },
```

Comment:
Can you define a constant for "Iterations"

Also the name for this variable is a bit ambiguous e.g. is it the maximum iterations, or completed iterations or something  else.

Reply:
Will change to "ModelIterationsCompleted". It's the number of round trips to the model that have occurred for the request.